src/applications/personalization/dashboard/components/cerner-widgets.js

```diff
@@ -3,11 +3,12 @@ import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox
 
 import { mhvUrl } from 'platform/site-wide/mhv/utilities';
 import { getCernerURL } from 'platform/utilities/cerner';
+import { externalRedirects } from 'platform/user/authentication/utilities';
 
 // Returns an AlertBox to present the user with info about working with the
 // Cerner facility they are enrolled at. Props allow you to edit a small amount
 // of the content that is rendered in the AlertBox.
-const CernerAlertBox = ({
+export const CernerAlertBox = ({
```

Comment:
does this need to be exported?

Reply:
Nope! Good catch - I had initially used this CernerAlertBox directly, but then realized I needed to wrap it with some ID for flexibility.